main.css

```diff
@@ -78,137 +86,138 @@
   border-width: 0.2px; */
 }
 
-#footer-form p {
-    font-size: 14px;
-}
+#col-3 {
+    position: absolute;
+    left: 1400px;
+    top: 30px;
```

Comment:
`HINT`

Using absolute values here to position the main content of the page may affect the display of the page. :thinking: Use `float` of `flex` instead.